src/CarbonAware/test/extensions/EmissionsDataExtensionsTests.cs

```diff
@@ -75,7 +75,7 @@ public void RollingAverage_ReturnsExpectedZeroCases(int windowSize, int tickSize
         }
 
         // Act
-        var result = this.data.RollingAverage(TimeSpan.FromMinutes(windowSize), TimeSpan.FromMinutes(tickSize));
+        var result = this.data.RollingAverage(TimeSpan.FromMinutes(windowSize), default, default, TimeSpan.FromMinutes(tickSize));
```

Comment:
```suggestion
        var result = this.data.RollingAverage(TimeSpan.FromMinutes(windowSize), tickSize: TimeSpan.FromMinutes(tickSize));
```

Reply:
YES! @bderusha and I were wondering how we could specify arguments. I was doing the pythonic way, `paramName = value` and it didn't work. Thank you!